scripts/ci/generate_ci_script.py

```diff
@@ -41,14 +41,38 @@
 from ros_buildfarm.templates import expand_template
 
 
+class _BuildToolArgsHelper(argparse.Action):
+
+    def __call__(self, parser, namespace, values, option_string=None):
+        setattr(namespace, self.dest, values)
+        if (
+            getattr(namespace, 'rosdistro_name', None) is not None and
+            getattr(namespace, 'config_url', None) is not None and
+            getattr(namespace, 'ci_build_name', None) is not None
```

Comment:
Maybe rewrite this condition to something like the following for readability: `if None not in (getattr(...), getattr(...), getattr(...)):`.

Reply:
0d457c2